lib/membrane_http_adaptive_stream/sink.ex

```diff
@@ -370,7 +370,11 @@ defmodule Membrane.HTTPAdaptiveStream.Sink do
           Manifest.add_segment(
             manifest,
             track_id,
-            [complete?: false],
+            [
+              complete?: false,
+              duration: buffer.metadata.duration,
+              byte_size: byte_size(buffer.payload)
+            ],
```

Comment:
We didn't pass the buffer `duration` and `byte_size` when adding segment to manifest, so they were always 0 (default).